### DISCUSSION TITLE
can SAM be customized?

### DISCUSSION THREAD

**Discussion by frankool** [vietanhdev/anylabeling#64] [Q&A]
Number of points, shapes that are disjoint individual labels... for batch work

**Comment by vietanhdev** [comment]
Can you describe a situation when batch work can be useful?

**Comment by frankool** [comment]
This is only an example but i think that one of the main power of SAM is to create a lot of labels at the same time, and if it´s possible from example filter by features like Area they are usually simillar.  And to customize the model with number of points because depending in the  size of the objects to label more points works usually better.

Thanks one and thousand times for the works this is the most useful tool for labeling that i have seen ever.


![Screen
![basetst](https://user-images.githubusercontent.com/75984388/235359940-5cf99072-a7ca-42b1-85a0-6796c2342d97.jpg)

![multi](https://user-image
![basetst](https://user-images.githubusercontent.com/75984388/235359791-f013b346-fa64-422e-921e-10fd2b1f5330.jpg)
s.githubusercontent.com/75984388/235358802-5ca3e9c6-d35b-403d-b627-f658968d0504.jpg)
shot_2023-04-30-13-28-49-487_com android chrome](https://user-images.githubusercontent.com/75984388/235358373-b9e882ee-5fb1-4033-b2d3-b8a94f8e0dfe.jpg)
![Screenshot_2023-04-30-13-32-41-158_com android chrome](https://user-images.githubusercontent.com/75984388/235358381-7244df72-ecda-4ca9-9212-16eee89c9dda.jpg)
![Screenshot_2023-04-30-13-33-42-560-edit_com android chrome](https://user-images.githubusercontent.com/75984388/235358394-bdf8e663-5c0b-436b-89ff-ab52f943a36c.jpg)
![Screenshot_2023-04-30-13-23-55-442_com android chrome](https://user-images.githubusercontent.com/75984388/235358423-dd35062a-4e22-480f-ba04-4f49cb55f614.jpg)
![Screenshot_2023-04-30-13-24-40-762_com android chrome](https://user-images.githubusercontent.com/75984388/235358424-396a998f-dde7-454c-95fe-c84e62a4eaf2.jpg)
![Screenshot_2023-04-30-13-25-45-829_com android chrome](https://user-images.githubusercontent.com/75984388/235358426-c544897e-ba28-4fc0-9fc4-4a743451a950.jpg)
